README.md

```diff
@@ -1,17 +1,14 @@
 # My porfolio
-> In this project i built a personal portfolio adding a contact form and making the project to 
-be responsive on both mobile and desktop version  . no  complex coding was done here 
-
+> In this project i built a personal portfolio adding a contact form and making the project to
+be responsive on both mobile and desktop version. no complex coding was done here .
 ## Built With
 
 - HTML
 - CSS
 - Linters
 
 ## Live Demo (if available)
-
-[Live Demo Link](https://livedemo.com)
-
+(https://ngufor-emmanuella.github.io/first-micro-project)
```

Comment:
- [ ] **[OPTIONAL]** You can format your markdown to look better by doing something like so `[Live Demo](https://ngufor-emmanuella.github.io/first-micro-project)` just as you have done with your author links below.